content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,354 @@
+---
+title: VMAX
+menuTitle: VMAX 
+description: "VMAX Support Matrix"
+weight: 19
+disableToc: false
+tags: ["support matrix", "delfin", "drivers", "dell emc"]
+---
+
+### Model and Management software versions supported
+
+<table>
+   <thead>
+      <tr>
+         <th>Models</th>
+         <th>Management Software Versions</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>VMAX 250F, VMAX 450F, VMAX 850F, VMAX 950F, VMAX 100K, VMAX 200K,VMAX 400K</td>
+         <td>Unisphere 9.0 or later</td>
+      </tr>
+   </tbody>
+</table>
+
+### Access information to register storage device
+
+<table>
+   <thead>
+      <tr>
+         <th>Access Type</th>
+         <th>Attributes</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td rowspan="4">REST</td>
+         <td>Host</td>
+         <td>Unishpere host name</td>
+      </tr>
+      <tr>
+         <td>Port</td>
+         <td>Unisphere port name</td>
+      </tr>
+      <tr>
+         <td>Username</td>
+         <td>Username</td>
+      </tr>
+      <tr>
+         <td>Password</td>
+         <td>Password</td>
+      </tr>
+   </tbody>
+</table>
+
+### Alert management information
+
+<table>
+   <tbody>
+      <tr>
+          <th>SNMP trap processing</th> 
+          <td>Supported</td> 
+      </tr>
+      <tr>
+         <th>Query backend alert</th>
+         <td>Supported</td>
+      </tr>
+      <tr>
+         <th>Clear backend alert</th>
+         <td>Supported</td>
+      </tr>
+      <tr>
+         <th>SNMP versions for trap</th>
+         <td>V1</td>
+      </tr>
+      <tr>
+         <th>SNMP trap source</th>
+         <td>Solution Enabler</td>
+      </tr>
+      <tr>
+         <th>Query alert source</th>
+         <td>Unisphere</td>
+      </tr>
+   </tbody>
+</table>
+
+### Performance registration information
+
+<table>
+   <thead>
+      <tr>
+         <th>Resource</th>
+         <th>Default Interval</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
```

Comment:
@Shruthi-1MN Table is empty. remove if not available or applicable.

Reply:
@NajmudheenCT has to confirm